systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -1029,8 +1030,76 @@ void checkContainerConfiguration(String podNamePrefix, String containerName, Str
             pods.forEach(pod -> {
                 pod.getSpec().getContainers().stream().filter(c -> c.getName().equals(containerName))
                         .forEach(container -> {
+                            List list = container.getEnv().stream().map((Function<EnvVar, Object>) EnvVar::getName).collect(Collectors.toList());
+                            assertTrue(list.contains(configKey));
+
                             container.getEnv().stream().filter(envVar -> envVar.getName().equals(configKey))
```

Comment:
Do we really need to go through the env vars twice? First to collect all the names and then to check value? Wouldn't doing it once ith a filter be enough?